server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/eml/twoAttachments.eml

```diff
@@ -24,7 +24,7 @@ Content-Type: multipart/mixed;
 Content-Type: text/html; charset=utf-8; format=flowed
 Content-Transfer-Encoding: 8bit
 
-<b>html</b>
+<b>html tiramisu</b>
```

Comment:
For a reason I can't explain all my test worked when I was searching for "html" maybe it is a ES stopword or something like that.